app/models/pawn.rb

```diff
@@ -7,4 +7,40 @@ def unicode_symbol
       return "&#9823;"
     end
   end
+
+  def valid_move?(destination_x, destination_y)
+    valid = super(destination_x, destination_y)
+    if valid
+      if (destination_x - self.x_position).abs > 0 || (destination_y - self.y_position).abs > 1
+        valid = false
+      end
+    end
+    return valid
```

Comment:
It is within the def valid_move? and end, so I keep it in the method. I think that code is being called after the first two if statements -- but now that you bring it up, maybe not. I'll refactor to include with the if statements.